### DISCUSSION TITLE
PID tuning can have wind resistance?

### DISCUSSION THREAD

**Discussion by sung-wook-kim** [iNavFlight/inav#7362] [Q&A]
Can wind resistance be achieved with only pid tuning without information on cruising speed and wind speed at more than 3 meters per second?

**Comment by DzikuVx** [comment]
You have to explain what you understand as "wind resistance"

**Reply by sung-wook-kim** [reply]
During the Gps hold, I felt that the pid control could not be held in the target position if the wind continued to blow. I described the ability to maintain a target position despite strong winds as wind resistance. I wonder if you think you can get this ability by pid tuning.

thank you

**Reply by avsaase** [reply]
@sung-wook-kim INAV's nav PID controller should be able to hold position even during high winds. When there are a lot of gusts the quad (I assume you are talking about quads here) is bound to get blown from its position before it returns to it. 

Can you upload some DVR and/or a log so we can see whats's going on? There are several factors that could cause bad position hold precision, notably magnetic interference of the compass.
